ui/infra-compare/InfraCompareTableView.jsx

```diff
@@ -103,7 +103,7 @@ export default class InfraCompareTableView extends React.Component {
     }
 
     const tableNames = [
-      ...new Set(data.map((item) => item.job_type__name)),
+      ...new Set(data.map((item) => item.job_type__name.replace(/-\d+$/, ''))),
```

Comment:
@jmaher I am splitting the jobNames here before storing them in the tableNames.
This will help us not to calculate data for the same jobName with chunk removed. This even solves the last bug of aggregating the jobNames for us.
So it will match the jobName as you mentioned [here](https://github.com/mozilla/treeherder/pull/6545#discussion_r444487417)